### PR TITLE
Implement pagination in Contents

### PR DESCRIPTION
**Pull request by KonradBauer**
I implement pagination in Contents (Popular movies and popular peoples). Check look and responsiveness

### CODE REVIEW

File: src/layouts/Pagination/styled.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ export const Box = styled.div`
   justify-content: center;
   flex-direction: row;
   align-items: center;
-  padding: 0px;
+  padding-bottom: 109px;
```
Comment (piotrpaczuski) [inline src/layouts/Pagination/styled.js:8]: Padding is reset in the mobile view because it is set padding to 0px on each side.

**Review by piotrpaczuski — APPROVED** [review]
All looks correctly, but you must repair paddings on mobile.

File: src/layouts/Pagination/styled.js (hunk 1 above)
Comment (KonradBauer) [reply src/layouts/Pagination/styled.js:8]: Padding now is set to 31px to bottom, like in Figma project.

**Review by florekdeveloper — APPROVED** [review]
Pagination looks ok now.

**Pull request merged**